src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -163,6 +171,33 @@ FixedwingAttitudeControl::parameters_update()
 	param_get(_parameter_handles.trim_roll, &(_parameters.trim_roll));
 	param_get(_parameter_handles.trim_pitch, &(_parameters.trim_pitch));
 	param_get(_parameter_handles.trim_yaw, &(_parameters.trim_yaw));
+	param_get(_parameter_handles.trim_roll_vmin, &(_parameters.trim_roll_vmin));
+	param_get(_parameter_handles.trim_roll_vmax, &(_parameters.trim_roll_vmax));
+
+	if (fabsf(_parameters.trim_roll_vmin) < 0.01f && fabsf(_parameters.trim_roll_vmax) < 0.01f) {
+		_parameters.trim_roll_vmin = _parameters.trim_roll;
+		_parameters.trim_roll_vmax = _parameters.trim_roll;
+	}
+
+	param_get(_parameter_handles.trim_pitch_vmin, &(_parameters.trim_pitch_vmin));
+	param_get(_parameter_handles.trim_pitch_vmax, &(_parameters.trim_pitch_vmax));
+
+	if (fabsf(_parameters.trim_pitch_vmin) < 0.01f && fabsf(_parameters.trim_pitch_vmax) < 0.01f) {
+		_parameters.trim_pitch_vmin = _parameters.trim_pitch;
+		_parameters.trim_pitch_vmax = _parameters.trim_pitch;
+	}
+
+	param_get(_parameter_handles.trim_yaw_vmin, &(_parameters.trim_yaw_vmin));
+	param_get(_parameter_handles.trim_yaw_vmax, &(_parameters.trim_yaw_vmax));
+
+	if (fabsf(_parameters.trim_yaw_vmin) < 0.01f && fabsf(_parameters.trim_yaw_vmax) < 0.01f) {
+		_parameters.trim_yaw_vmin = _parameters.trim_yaw;
```

Comment:
I don't think you should mess with the param "value" like this. Put the check where you're adding the trim rather than trying to pass through the regular trim.